openssl-sys/Cargo.toml

```diff
@@ -25,7 +25,7 @@ bssl-sys = { version = "0.1.0", optional = true }
 [build-dependencies]
 bindgen = { version = "0.64.0", optional = true, features = ["experimental"] }
 cc = "1.0.61"
-openssl-src = { version = "111", optional = true }
+openssl-src = { version = "300.1.2", optional = true, features = ["legacy"] }
```

Comment:
Legacy feature is required for the tests at least.

Reply:
It does seem like we realistically probably want to enable this - thoughts @alex?

Reply:
We enable it by default in pyca/cryptography, its required for basically any encrypted key serialization (since they all use garbage algorithms).